community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/FunctionsAcceptanceTest.scala

```diff
@@ -412,6 +412,31 @@ class FunctionsAcceptanceTest extends ExecutionEngineFunSuite with NewPlannerTes
     result.toList should equal(List(Map("point" -> CartesianPoint(2.3, 4.5))))
   }
 
+  test("point function should work with integer arguments") {
+    val result = executeWithAllPlanners("RETURN point({x: 2, y: 4, crs: 'cartesian'}) as point")
+    result should useProjectionWith("Point")
+    result.toList should equal(List(Map("point" -> CartesianPoint(2, 4))))
+  }
+
+  test("should fail properly if missing cartesian coordinates") {
+    a [SyntaxException] shouldBe thrownBy(executeWithAllPlanners("RETURN point({params}) as point",
+                                                                 "params" -> Map("y" -> 1.0, "crs" -> "cartesian")))
+  }
+
+  test("should fail properly if missing geographic coordinates") {
+    a [SyntaxException] shouldBe thrownBy(executeWithAllPlanners("RETURN point({params}) as point",
+                                                                 "params" -> Map("y" -> 1.0, "crs" -> "WGS-84")))
+  }
+
+  test("should fail properly if unknown coordinate system") {
+    a [SyntaxException] shouldBe thrownBy(executeWithAllPlanners("RETURN point({params}) as point",
+                                                                 "params" -> Map("y" -> 1.0, "crs" -> "WGS-1337")))
```

Comment:
Shouldn't this test have an `x` coordinate? The failure condition is the invalid value of the `crs` parameter, isn't it?